src/chrome/app/manifest.json

```diff
@@ -23,5 +23,9 @@
     "tcpServer": { "listen": "" },
     "udp": { "send": "", "bind": "" }
   },
+  "externally_connectable": {
+    "ids" : ["pjpcdnccaekokkkeheolmpkfifcbibnj"],
+    "matches" : ["*://*.uproxy.org/*", "*://1-0-1-test-dot-uproxysite.appspot.com/*"]
```

Comment:
It complains when I try to do that:
Invalid match pattern '*://*-dot-uproxysite.appspot.com/*'

Looking at the rules online, I think * can only proceed a dot in the first part of the URL, but appspot.com is considered a TLD and so *.appspot.com isn't allowed.